service/volume.py

```diff
@@ -67,9 +103,47 @@ def create_volume(esh_driver, identity_uuid, name, size,
         metadata=metadata,
         snapshot=snapshot,
         image=image)
+
+    if not success and raise_exception:
+        raise exceptions.VolumeError("The volume failed to be created.")
```

Comment:
This error hides whether or not the failure was a result of "Us" (Atmosphere, bad process, etc.) or "Them" (Cloud failure, cloud error, etc.).. Perhaps we could split "ServiceException" into CloudService Vs. APIService and return a more appropriate error message + code based on that?